submissions/jonasMwansa/data structures and algorithms 1/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="index.js" type="text/javascript"></script>
```

Comment:
It is bad coding practice to have your script in the <head> tag.

Reply:
I have just noticed I placed it in the head tag,
let me place it at the bottom of the body, thank you @ToluEnny24